pkg/cosign/git/gitlab/gitlab.go

```diff
@@ -70,6 +71,7 @@ func (g *Gl) PutSecret(ctx context.Context, ref string, pf cosign.PassFunc) erro
 		EnvironmentScope: gitlab.String("*"),
 	})
 	if err != nil {
+		ui.Infof(ctx, "WARNING: if you want to use a custom Gitlab host, set the \"GITLAB_HOST\" environment variable.")
```

Comment:
maybe, we can say something like: "if you are using a self-hosted gitlab please set the GITLAB_HOST your server name"

thanks for this PR!

Reply:
thanks for suggestion, changed!